pages/docs/concepts/asyncapi-document/tags.md

```diff
@@ -0,0 +1,249 @@
+---
+title: 'Tags'
+weight: 63
+---
+
+--- Unique Description about AsyncAPI document to be inserted here ---
+
+A `tag` is a label or category that helps you logically group related objects such as channels or servers within an event-driven system. The `tags` object provide a structured way to group channels, operations or other components based on their functionality, purpose and/or any other relevant criteria.
+
+The `tags` object serves as a container for defining and managing tags in a systematic manner. Within the `tags` object, you can define individual tags, specifying a name and providing an optional description that adds further context to the tag's purpose or usage. 
+
+Once tags are defined within the `tags` object, they can be assigned to channels, operations or any other component using the tags property. By including the relevant tag names in the tags property, objects such as channels, servers or components are associated with the appropriate tags, grouping them based on their shared characteristics.
```

Comment:
> Once tags are defined within the `tags` object, they can be assigned

what do you mean by "they can be assigned"? Tags list is individual per component. Later tooling interpret that if you are consistent with Info-level tags and tags in operations, then there is some other special navigation in docs showing up. But on spec level, they are independent  Tags lists.

unless you mean and suggest that people should add Tags to `components` and then reference them in channels and other locations? 🤔 but I don't think so as you talk about it in another paragraph